kong/core/cluster.lua

```diff
@@ -63,33 +72,35 @@ local function async_autojoin(premature)
     if (autojoin_retries < ASYNC_AUTOJOIN_RETRIES) then
       create_timer(ASYNC_AUTOJOIN_INTERVAL, async_autojoin)
     end
+  elseif err ~= "exists" then
+    log_error(err)
```

Comment:
Doesn't the error print the file name that generated it and the line number? If it ever appears in the logs, we know where it is.